xml/docu_styleguide.language.xml

```diff
@@ -347,14 +357,13 @@
   <title>Tense</title>
   <para>
    Use the simple present tense. Apply the simple present tense even to
-   sentences with <quote>if</quote> or <quote>when</quote> clauses:
+   sentences with <quote>if</quote> or <quote>when</quote> clauses and to
+   prerequisites of an action:</para>
+  <para>
    <quote>If this happens, go there.</quote>
   </para>
   <para>
-   Prerequisites of an action should be expressed in the present tense as
-   well: <quote>Glibc is installed.</quote> In some cases, no verb is
-   necessary before the prerequisite of an action: <quote>a 1&nbsp;GHz
-   processor or better.</quote>
+   <quote>Glibc is installed.</quote>
   </para>
```

Comment:
The comment is about the structure you created:
> - para (Use the simple present tense. [...] of an action:)
> - para
>    - quote (If this happens, go there.)
> - para
>    - quote (Glibc is installed.)

Maybe this would work better (I am not 100% sure about the `..` at the end of this suggestion, improvements welcome):

```suggestion
  <para>
   Use the simple present tense. Apply the simple present tense even to
   sentences with <quote>if</quote> or <quote>when</quote> clauses and to
   prerequisites of an action. For example,
   <quote>If this happens, go there.</quote> or
   <quote>Glibc is installed.</quote>.
 <para>
```
Alternatively, it should be a list, but I am not sure this is warranted when the 2 items are just short examples.

Reply:
Opened a separate PR - see https://github.com/SUSE/doc-styleguide/pull/165